src/client/Microsoft.Identity.Client/PublicApi/net462/PublicAPI.Shipped.txt

```diff
@@ -148,6 +148,10 @@ const Microsoft.Identity.Client.MsalError.WebView2LoaderNotFound = "webview2load
 const Microsoft.Identity.Client.MsalError.WebView2NotInstalled = "webview2_runtime_not_installed" -> string
 const Microsoft.Identity.Client.MsalError.WebviewUnavailable = "no_system_webview" -> string
 const Microsoft.Identity.Client.MsalError.WsTrustEndpointNotFoundInMetadataDocument = "wstrust_endpoint_not_found" -> string
+const Microsoft.Identity.Client.MsalError.MissingTenantedAuthority = "missing_tenanted_authority" -> string
```

Comment:
nit: All the public API changes should go to the "unshipped.txt". And on release, you move them to shipped.txt

Reply:
moved it